src/components/LabCard/LabCard.js

```diff
@@ -0,0 +1,212 @@
+        return labResult.valueQuantity.code ? `${labResult.valueQuantity.value} ${labResult.valueQuantity.code}` : labResult.valueQuantity.value;
+      }
+    }
+    return labResult.valueString || '';
+  }
+
+  const displayDateTime = (dateTimeString) => {
+    const [date, time] = dateTimeString.split('T');
+    if (!time) return date;
+    const d = new Date(dateTimeString);
+    const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+    return (`${date} ${d.toLocaleTimeString('en-US')} (${timezone})`);
+  }
+
+  const displayReferenceRange = (referenceRange) => {
+    if (referenceRange.text) return referenceRange.text;
+    const lowVal = referenceRange.low;
+    const highVal = referenceRange.high;
+    if (lowVal && highVal) return (`${lowVal.value} ${lowVal.unit || ''} - ${highVal.value} ${highVal.unit || ''}`);
+    if (lowVal) return (`$>= ${lowVal.value} ${lowVal.unit || ''}`);
+    if (highVal) return (`$<= ${highVal.value} ${highVal.unit || ''}`);
+    return '';
+  }
+
+  const HealthCardLabResult = ({ labResult }) => (
+    <Box className={[styles.flexCol, styles.vaccinationRoot].join(' ')}>
+      <Box className={styles.divider}>
+        <Divider className={styles.line2} />
+      </Box>
+      <Grid container className={styles.grid}>
+        <Grid container item className={styles.gridRow} spacing={0}>
+          <Grid item xs={3} sm={2} className={styles.gridLabel}>
+            <Typography>{t('healthcarddisplay.Lab Test')}</Typography>
+          </Grid>
+          <Grid item xs={9} sm={10} className={styles.gridItem}>
+            <Typography>
+              <Box component="span" fontWeight="700">
+                {displayLabTestCode(labResult.code?.coding[0])}
+              </Box>
+            </Typography>
+          </Grid>
+        </Grid>
+
+        <Grid container item className={styles.gridRow} spacing={0}>
+          <Grid item xs={3} sm={2} className={styles.gridLabel}>
+            <Typography>{t('healthcarddisplay.Date')}</Typography>
+          </Grid>
+          <Grid item xs={9} sm={10} className={styles.gridItem}>
+            <Typography className={styles.date}>
+              {displayDateTime(labResult.effectiveDateTime)}
+            </Typography>
+          </Grid>
+        </Grid>
+
+        <Grid container item className={styles.gridRow} spacing={0}>
+          <Grid item xs={3} sm={2} className={styles.gridLabel}>
+            <Typography>{t('healthcarddisplay.Test Result')}</Typography>
+          </Grid>
+          <Grid item xs={9} sm={10} className={styles.gridItem}>
+            <Typography>
+              {displayLabTestResult(labResult)}
+            </Typography>
+          </Grid>
+        </Grid>
+
+        <Grid container item className={styles.gridRow} spacing={0}>
+          <Grid item xs={3} sm={2} className={styles.gridLabel}>
+            <Typography>{t('healthcarddisplay.Ref. Range')}</Typography>
```

Comment:
We talked a little about this in the working session - should we only display the field names if they are populated? ex. if no reference range value is included in the card, then don't display "Ref. Range" at all